### ISSUE TITLE
MissingAttributeError for image_description in api home_controller.rb

### ISSUE THREAD

**Issue by lutoma** [mastodon/mastodon#26333]
### Steps to reproduce the problem

1. Run Mastodon from main branch (commit 04e7efac3f2ebb4336c48335720611120eae19ba)
2. Open local/federated timeline and scroll down to load new posts

### Expected behaviour

New posts get loaded

### Actual behaviour

Internal Server Error

### Detailed description

Traceback:

```
FATAL -- : [86768a25-be09-4501-9f7a-6180adfff631]
ActiveModel::MissingAttributeError (missing attribute: image_description):

app/controllers/api/v1/timelines/home_controller.rb:14:in `show'
app/controllers/concerns/localized.rb:11:in `set_locale'
lib/mastodon/rack_middleware.rb:9:in `call'
```

Reverting to a commit before 8da99ffb0dcf5012179e4142799ff473910f5a66 (and rolling back associated migrations) fixes the issue.

### Mastodon instance

ohai.social

### Mastodon version

main (04e7efac3f2ebb4336c48335720611120eae19ba)

### Technical details

- Ruby version: 3.2.2
- Node.js version: 16.20.1

**Comment by renchap** [comment]
Did you forget to run migrations?

**Comment by lutoma** [comment]
I ran migrations using `RAILS_ENV=production DB_PORT=5432 bundle exec rails db:migrate` and then also verified they had applied using `RAILS_ENV=production bundle exec rails db:migrate:status`.

**Comment by ClearlyClaire** [comment]
Can you update again, see if this is still happening, and if it is, run `tootctl cache clear`?

**Comment by lutoma** [comment]
After updating to 04e7efac3f2ebb4336c48335720611120eae19ba again and running the migration I ended up with the same errors. But clearing the cache with `tootctl cache clear` did indeed fix the issue and everything seems to be working properly now. Thanks!

**Comment by ClearlyClaire** [comment]
Thanks for confirming my hunch!

**Comment by ClearlyClaire** [comment]
Fixed by #26343
